typescript/infra/scripts/send-random-messages.ts

```diff
@@ -0,0 +1,178 @@
+import { Wallet } from 'ethers';
```

Comment:
This script is a conversion of the Kathy hardhat task into a simpler script. 
The reason for the change is that HH configs must only be CJS but tasks must be defined in the HH configs. Give that this didn't need HH in the first place, it's now just a script

Reply:
Curious how this would surface. Thanks for the refactor